src/Constants.iOS.cs.in

```diff
@@ -143,5 +143,6 @@ namespace MonoTouch {
 		public const string LinkPresentationLibrary = "/System/Library/Frameworks/LinkPresentation.framework/LinkPresentation";
 		public const string VisionKitLibrary = "/System/Library/Frameworks/VisionKit.framework/VisionKit";
 		public const string CoreHapticsLibrary = "/System/Library/Frameworks/CoreHaptics.framework/CoreHaptics";
+		public const string MetricKitLibrary = "/System/Library/Frameworks/MetricKit.framework/MetricKit";
```

Comment:
please keep them sorted to ease comparison with the other files

Reply:
Not much of a sort order here heh, I guess @mandel-macaque  can do the hardwork sort here :P